File: tests/indexing/parsing/lang/fixtures/go/BodySizes.go
package example

func emptyBody() {}

func withLogic(items []int) (int, error) {
	if len(items) == 0 {
		return 0, nil
	}
	max := items[0]
	for _, item := range items[1:] {
		if item > max {
			max = item
		}
	}
	return max, nil
}

File: tests/indexing/parsing/lang/fixtures/php/BodySizes.php
<?php

abstract class Repository
{
    abstract public function abstractMethod(): void;

    public function emptyBody(): void
    {
    }

    public function withLogic(array $items): int
    {
        $total = 0;
        foreach ($items as $item) {
            if ($item > 0) {
                $total += $item;
            }
        }
        return $total;
    }
}
